demo/timing.c

```diff
@@ -10,10 +10,10 @@
 #define SLEEP
 #endif
 
-#ifdef LTM_TIMING_REAL_RAND
-#define LTM_TIMING_RAND_SEED  time(NULL)
+#ifdef LTM_TIMP_MING_REAL_RAND
```

Comment:
I've the impression your `MP_MIN` regex went wild ;)

Reply:
haha, thanks for checking :)